docs/spec_draft.md

```diff
@@ -358,6 +359,56 @@ IEEE-754 specification.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.concatenate
+
+### Semantics
+
+Concatenates sequence of tensors in `val` along dimension `dimension` and
+produces a tensor as a result. The tensors are of the same rank as the input
+operands except the specified dimension.
+
+### Operands
```

Comment:
Oh, one more thing. Let's change the order of the items in this section to follow the order of definitions in ODS, i.e. `val` first and `dimension` second?